nautobot/dcim/views.py

```diff
@@ -2455,7 +2506,7 @@ def queryset(self):  # pylint: disable=method-hidden
         return Device.objects.annotate_config_context_data()
 
 
-class DeviceChangeLogView(ObjectChangeLogView):
+class DeviceChangeLogView(ObjectChangeLogView, DevicePageMixin):
```

Comment:
Can this be changed for self-consistency, or does this not work for some reason (in which case please add a comment explaining this)

```suggestion
class DeviceChangeLogView(DevicePageMixin, ObjectChangeLogView):
```